static/js/click_handlers.js

```diff
@@ -46,6 +46,12 @@ import * as user_status_ui from "./user_status_ui";
 import * as util from "./util";
 
 export function initialize() {
+    // LOGIN BUTTON
+
+    $("body").on("click", ".login_button", () => {
+        window.location.href = "/login/";
```

Comment:
This probably should have stopPropagation/preventDefault, like essentially all click handlers.